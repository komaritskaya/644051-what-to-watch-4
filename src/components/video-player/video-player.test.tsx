import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player';

const movie = {
  id: `123`,
  title: `Some Film`,
  poster: `some-poster.jpg`,
  genre: `comedy`,
  year: 2000,
  director: `Some Director`,
  cast: [`Actor One`, `Actor Two`],
  cover: `some-bg-poster.jpg`,
  trailer: `some-trailer.mp4`,
  description: `Damn good film`,
  rating: 10.0,
  reviewsCount: 100,
};

it(`VideoPlayer is rendered correctly`, () => {
  const tree = renderer.create(<VideoPlayer
    isPlaying={false}
    poster={movie.poster}
    src={movie.trailer}
  />, {
    createNodeMock: () => {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
