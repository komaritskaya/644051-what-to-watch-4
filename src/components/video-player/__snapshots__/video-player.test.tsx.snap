// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer is rendered correctly 1`] = `
<video
  className="player__video"
  height="175"
  muted={true}
  poster="img/some-poster.jpg"
  src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
  width="280"
/>
`;
